integrations/jira/.port/resources/blueprints.json

```diff
@@ -1,4 +1,83 @@
 [
+  {
+    "identifier": "jiraBoard",
+    "title": "Jira Board",
+    "description": "This blueprint represents a Jira board",
+    "icon": "Jira",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Board URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the board in Jira"
+        },
+        "type": {
```

Comment:
If we can make this field an enum it will be great. But that's if you know all the possible board types. Checking here, I see Scrum and Kanban board

https://www.atlassian.com/software/jira/guides/boards/overview#types-of-boards